functions/tips/index.js

```diff
@@ -94,8 +94,15 @@ const httpsAgent = new https.Agent({keepAlive: true});
  */
 exports.connectionPooling = async (req, res) => {
   try {
-    const {data} = await fetch('/', {httpAgent, httpsAgent});
-    res.status(200).send(`Data: ${data}`);
+    const url = 'https://www.example.com/';
+
+    // Select the appropriate agent to use based on the URL.
+    const agent = url.includes('https') ? httpsAgent : httpAgent;
```

Comment:
This constant could probably be removed by inlining it.

Reply:
I wanted to call it out explicitly, which is why I didn't do that.